app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -149,33 +213,143 @@
   }
 
   .accordion dd > .content {
-    padding: 0 0.935rem;
+    padding: 0;
+  }
+
+  .accordion dd.accordion-navigation {
+    margin-top: 2rem;
   }
 
   .actions {
-    border: 1px solid #CCCCCC;
+    margin-top: 1rem;
 
     .action {
-      padding: 10px 40px;
-      border-top: 2px solid #333333;
+      padding: 1.5rem;
+      border-top: 1px solid rgb(148, 148, 148);
     }
   }
 
   /* Stages in completed guides */
-  .stages{
-    .image img{
+  .stages {
+
+    // This is shared with guide creation steps, and should be DRYed up
+
+    .stage-wrapper {
+      display: inline-block;
+      overflow: hidden;
+      height: 300px;
+      margin-bottom: 3%;
+    }
+
+    .stage-option {
+      display: block;
+      border-radius: 4px;
+      padding: .25rem;
+      border: solid 1px #DFDFDF;
+      width: 100%;
+      height: 100%;
+    }
+
+    .stage-option, .chosen-stage > .bubble {
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-position: center center;
+      background-image: url('stages/other_sm.png');
+
+
+      &.Preparation {
+        background-image: url('stages/prep_sm.png');
+      }
+
+      &.Sow {
+        background-image: url('stages/sow_sm.png');
+      }
+
+      &.Germination {
+        background-image: url('stages/germination_sm.png');
+      }
+
+      &.Seedling {
+        background-image: url('stages/sprouting_sm.png');
+      }
+
+      &.Juvenile {
+        background-image: url('stages/juvenile_sm.png');
+      }
+
+      &.Adult {
+        background-image: url('stages/adult_sm.png');
+      }
+
+      &.Flowering {
+        background-image: url('stages/flowering_sm.png');
+      }
+
+      &.Fruit {
+        background-image: url('stages/fruiting_sm.png');
+      }
+
+      &.Dormant {
+        background-image: url('stages/dormant_sm.png');
+      }
+    }
+
+    // End need DRYing
+
+    .action-content {
+      margin-top: 1rem;
+    }
+
+    .action-content {
+      border-bottom: #CDCDCD 2px solid;
+    }
+
+    .action-content:last-child {
+      border-bottom: none;
+    }
+
+    .image img {
       height: 170px;
       background-size: cover;
       background-position: center;
       width: 100%;
     }
 
-    h3{
+    h3 {
       font-size: 1rem;
       margin: 0;
     }
 
-    /* Action labels in completed guides look different */
+    .tabs {
+      background-color: #efefef;
+    }
+
+    .tabs-content {
+      background-color: #f6f6f6;
+    }
+
+    .tabs dd > a {
```

Comment:
Merge rule `.tabs dd > a` with rule on line 323<br>Selector should have depth of applicability no greater than 2, but was 5